### DISCUSSION TITLE
Crazyflie Client crashes when "estimate geometry" with Lighthouse positioning

### DISCUSSION THREAD

**Discussion by jakob-schreiber** [bitcraze/discussions#1595] [Q&A]
When I follow the wizard on setting up the lighthouse positioning system, the client crashed once I click on "estimate geometry" and this error is shown: bus error  /usr/bin/python3 -m cfclient.gui 

Can someone help me

**Comment by gemenerik** [comment]
I see this discussion is closed, did you fix the error somehow?
